### ISSUE TITLE
Editing project information is not saving

### ISSUE THREAD

**Issue by in-op** [aweber/imbi#41]
Currently editing project information on the Overview tab of a project is not saving any edits made.

The console shows a 500 error to the PATCH endpoint:

```
PATCH https://imbi.aweber.io/projects/389 500 (Internal Server Error)
```

**Comment by joshbenner** [comment]
Related error from Imbi log:
```json
{
  "name": "tornado.application",
  "levelname": "ERROR",
  "asctime": "2021-03-26T15:04:43.879931+0000",
  "processName": "MainProcess",
  "message": "Uncaught exception PATCH /projects/389 (1.2.3.4)\nHTTPServerRequest(protocol='https', host='imbi.example.com', method='PATCH', uri='/projects/389', version='HTTP/1.1', remote_ip='1.2.3.4')",
  "module": "web",
  "exc_info": [
    "Traceback (most recent call last):\n",
    "  File \"/usr/local/lib/python3.9/site-packages/tornado/web.py\", line 1704, in _execute\n    result = await result\n",
    "  File \"/usr/local/lib/python3.9/site-packages/imbi/endpoints/base.py\", line 294, in patch\n    await self._patch(kwargs)\n",
    "  File \"/usr/local/lib/python3.9/site-packages/imbi/endpoints/base.py\", line 392, in _patch\n    result = await self.postgres_execute(\n",
    "  File \"/usr/local/lib/python3.9/site-packages/sprockets_postgres.py\", line 686, in postgres_execute\n    return await connector.execute(\n",
    "  File \"/usr/local/lib/python3.9/site-packages/sprockets_postgres.py\", line 192, in execute\n    return await self._query(\n",
    "  File \"/usr/local/lib/python3.9/site-packages/sprockets_postgres.py\", line 249, in _query\n    await method(**kwargs)\n",
    "  File \"/usr/local/lib/python3.9/site-packages/aiopg/cursor.py\", line 110, in execute\n    self._impl.execute(operation, parameters)\n",
    "  File \"/usr/local/lib/python3.9/site-packages/psycopg2/extras.py\", line 251, in execute\n    return super(RealDictCursor, self).execute(query, vars)\n",
    "KeyError: 'archived'\n"
  ]
}
```

**Comment by ryangsteele** [comment]
I think it would also be nice to have the "500: Internal Server Error" message show up in a more noticeable way. Currently, it's not very noticeable and I missed it several times:

![Screen Shot 2021-03-26 at 11 39 03 AM](https://user-images.githubusercontent.com/5678706/112656610-f3cb7c80-8e27-11eb-95a1-7aa1424cb64d.png)

**Comment by gmr** [comment]
I *think* that is a build issue with tree shaking, here's how it shows up in dev:
![Screen Shot 2021-03-26 at 11 42 08 AM](https://user-images.githubusercontent.com/49469/112656966-50c73280-8e28-11eb-9d6c-7232f4555442.png)

**Comment by gmr** [comment]
Few observations when recreating:

a 304 will prevent project links from being saved

**Comment by gmr** [comment]
Indeed for the error message issue, the alert component relies on css classes defined in src/css/custom-compoents.css and is not being included during build.

**Comment by gmr** [comment]
Fix pending for the 500 Error

**Comment by gmr** [comment]
CSS issue will be addressed as #44

**Comment by gmr** [comment]
Fixed in main, will be released in 0.9.0
